content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+    The Code of Conduct outlines expected behaviour for all members of
+    the community at any event affiliated with the undergraduate
+    committee. Any member of the committee may prpose an amendment to
+    the CoC. The committee must ratify any proposed amendments to the
+    Code of Conduct.
+
+2.  Machine Usage Agreement\
+    The Machine Usage Agreement outlines expected conduct when using any
+    machines maintained by the undergraduate committee. The machine
+    usage agreement is maintained by the systems administrator. Any
+    proposed changes must be first ratified by wics-sys, followed by the
+    entire committee.
+
+3.  Office Poliy\
+    Office Policy outlines expected behaviour in the WiCS Undergraduate
+    Committee office, M3 1003. The office policy is maintained by the
+    Office Manager. Any proposed changes must be ratified by the
+    committee.
+
+## Succession Procedure ##
+
+1.  No later than the end of the first week of the fourth month of the
+    current term, a call for committee members should be announced on
+    the website and sent out to all affiliated mailing lists.
+
+2.  New volunteers caan apply to be on the committee for the following
+    term, and the current committee will select applicants for
+    interviews. A new volunteer shall be anyone who has never served on
+    the committee before, or has not served on the committee for 4 or
+    more terms. Interviewers should be chosen by the current committee.
+    If an interviewer has a conflict of interest with the interviewee,
+    an alternate interviewer will be chosen. In any case, if no
+    interviewer is explicitly chosen, the current Chair shall conduct
+    interviews. The interviews are intended to welcome the new member to
+    the committee, give them background on the committee's activities,
+    and help them find a role for the next term. The current committee
+    will ultimately select the comoposition of the committee for next
+    term.
+
+3.  Existing committee members can nominate themselves for positions on
+    the next term's committee given the following conditions:
+
+    -   No member may serve more than three consecutive terms;
+
+    -   No member may serve as Chair for more than one term
+        consecutively;
+
+    -   No member intending to serve as Chair or Secretary may be on
+        co-op;
+
+    -   Members wishing to nominate themselves for chair should be a
+        previous WiCS committee member. If no such volunteer exists, the
+        position opens up to new members;
+
+    -   Excluding that of Chair, no positions may be chosen until the
+        first official meeting of the new term's committee;
+
+    -   Existing committee members may compose no more than 75% of the
+        new committee, if more existing members are nominated than 75%
+        can accomodate, each returning member will submit a new
+        application to be reviewed by the serving Director of the WiCS
+        Committee.
+
```

Comment:
The Chair for the upcoming term's committee shall be selected using the following procedures:

- Returning members wishing to serve as Chair will be given preference over new members and must nominate themselves;
- If there are any nominees, the current committee may select and ratify one of the nominees as the Chair for the next term;
- If no returning member is chosen to serve as Chair, the position will open to new members and shall be selected by the committee in the upcoming term.

Note that I said "may" because it's possible the current committee could have an objection to a particular returning member serving as chair and so they should not be forced into selecting them simply on the basis of seniority.